### ISSUE TITLE
Response 404 from token page 

### ISSUE THREAD

**Issue by wci-aw** [suurjaak/Skyperious#51]
Hi support team,
I need your help - while I try to export messages to file, I need to input my skype password for getting images. Then I get an failure shown. "response 404 from token page - try again? 
Unfortunate the images within the chat are not downloaded into the html file.
Find here the screen shot - http://screencast.com/t/N8WtEFn9Lrq

Looking forward to your help.

Alex

**Comment by suurjaak** [comment]
Looks like Skype has changed things on their server side.

I will look into it. Thank you for reporting this.

**Comment by g4269159** [comment]
Hi, any luck with this one?

**Comment by wci-aw** [comment]
No still the same issue
Message: 
"Response 404 from token page" 
Try again?

It seems your software needs an update...

Regards, Alex

**Comment by g4269159** [comment]
Yeah, I've got the same issue, I was just wondering. I hope it's solved soon, it's really annoying.

**Comment by tewe** [comment]
It looks like you now need to log in for every image individually. It's possible to adjust Skyperious, but the photos will still be lower resolution than what you get in the Skype app, so I for one gave up. Microsoft officially doesn't want 3rd party clients.

**Comment by macik** [comment]
Get the same error message.

What about getting some stuff from
`media_messaging\media_cache_v3` folder?

**Comment by macik** [comment]
By the way — found «quick» solution to get images back to exported HTML.

1. export HTML «as is» (with error message). 
   Inspect generated HTML to locate link like
  ```
   https://api.asm.skype.com/s/i?0-neu-d1-c02e1a783b30b90a815d7c8a70037c67
   ```
   or 
   ```
   https://login.skype.com/login/sso?go=xmmfallback?pic=0-neu-d3-d060f58d9d05aefb6816e052c44cc80d
   ```
   It's link to view images online.

2. Change content of HTML to replace links to direct images with RegExp:

   **find pattern**: 
   ```
   (?:xmmfallback\?pic=|s/i\?)([\w\-]+)</a>
   ```
   **replace pattern**: 
   ```
   $0 <br/> <a href="https://api.asm.skype.com/v1/objects/$1/views/imgpsh_fullsize" target="_blank"><img src="https://api.asm.skype.com/v1/objects/$1/views/imgpsh_fullsize" style="width:300px;"></a>
   ```

3. View your HTML page in browser. If you still can't see images — just login to WebSkype ( https://login.skype.com/login ). And reload HTML. You should see images right in «chat stream» (page). 

4. Save page with loaded images (press Ctrl-S in Chrome or FireFox browser) as local copy.
5. Now you had locally your HTML and images.

![image](https://cloud.githubusercontent.com/assets/1009926/16842894/be48ac1a-49e7-11e6-9c9f-fd04be01cd09.png)

**Comment by arun-gurung** [comment]
I also tried to export the chats to HTML and it asks for the password all the time with an error. In the terminal, the error goes like this:

swig/python detected a memory leak of type 'wxPasswordEntryDialog *', no destructor found.

**Comment by suurjaak** [comment]
Closing this issue, as the Skype login page is no longer available, and login functionality has been removed in the latest version, Skyperious 3.6.

**Comment by suurjaak** [comment]
FYI: I have released a new version of Skyperious, v4.0, with login and shared image download functionality restored.

**Comment by arun-gurung** [comment]
Thank you.
